### PR TITLE
useConnectionTimeoutAsAcquisitionTimeout

### PR DESCRIPTION
**Pull request by xinlian12**
By default changing to use connectionTimeout as the rntbd acquisitionTimeout. 

But can override it by using system property: 
`        System.setProperty("COSMOS.TCP_CONNECTION_ACQUISITION_TIMEOUT_IN_MS", String.valueOf(overrideDuration.toMillis()));`

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM Thanks

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java [hunk 1]
```diff
@@ -169,6 +169,8 @@ public class Configs {
 
     public static final int MIN_MAX_RETRIES_IN_LOCAL_REGION_WHEN_REMOTE_REGION_PREFERRED = 1;
 
+    public static final String TCP_CONNECTION_ACQUISITION_TIMEOUT_IN_MS = "COSMOS.TCP_CONNECTION_ACQUISITION_TIMEOUT_IN_MS";
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java:172]: Should we also add the default value here?

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java:172]: no - intentionally left it out -> the default value will coming from the connectionTimeout config from the conenctionPolicy

**Pull request merged**